Meta/Lagom/ReadMe.md

```diff
@@ -1,43 +1,64 @@
 # Lagom
 
-The Serenity C++ library, for other Operating Systems.
+The Serenity C++ libraries, for other Operating Systems.
 
 ## About
 
 If you want to bring the comfortable Serenity classes with you to another system, look no further. This is basically a "port" of the `AK` and `LibCore` libraries to generic \*nix systems.
 
 *Lagom* is a Swedish word that means "just the right amount." ([Wikipedia](https://en.wikipedia.org/wiki/Lagom))
 
+Lagom is used by the Serenity project in the following ways:
+
+- [Build tools](./Tools) required to build Serenity itself using Serenity's own C++ libraries are in Lagom.
+- [Unit tests](../../Documentation/RunningTests.md) in CI are built using the Lagom build for host systems to ensure portability.
+- [Continuous fuzzing](#fuzzing-on-oss-fuzz) is done with the help of OSS-fuzz using the Lagom build.
+- [The Ladybird browser](../../Ladybird/README.md) uses Lagom to provide LibWeb and LibJS for non-Serenity systems.
+- [ECMA 262 spec tests](https://libjs.dev/test262) for LibJS are run per-commit and tracked on libjs.dev.
```

Comment:
Can you also make the "root" libjs.dev a link at least once?

Reply:
good point. updated the links at the end of each line